apollo-federation/src/supergraph/mod.rs

```
@@ -747,6 +766,45 @@ fn extract_object_type_content(
             )?;
         }
 
+        if let Some((context_spec_def, name_in_supergraph)) = context {
+            for directive in type_.directives.get_all(name_in_supergraph.as_str()) {
+                if let Some(context_name) =
+                    FederationSpecDefinition::context_directive_arguments(directive).ok()
```

Similar question as before. What kind of errors are being suppressed here?